xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -1299,7 +1296,7 @@ public void serverSideListenerFound() throws InvalidProtocolBufferException {
     ClientXdsClientTestBase.DiscoveryRpcCall call =
         startResourceWatcher(LDS, LISTENER_RESOURCE, ldsResourceWatcher);
     Listener listener =
-        buildListenerWithFilterChain(
+            (Listener)mf.buildListenerWithFilterChain(
```

Comment:
Isn't this cast redundant? Dealing with `Message` is enough.

Reply:
I have changed the decl to Message and only put the cast in the call to `fromEnvoyProtoListener`.